R/tspa.R

```diff
@@ -113,27 +114,50 @@
 
 
 tspa <- function(model, data, reliability = NULL, se = NULL,
-                 vc = NULL, cross_loadings = NULL, ...) {
+                 fsT = NULL, fsL = NULL, ...) {
+
+  if (nchar(model) == 0) {
+    stop("Please provide a structural path model.")
+  }
+
   if (!is.null(reliability)) {
     stop("tspa() currently does not support reliability model")
   }
+
   if (!is.data.frame(se)) {
     se <- as.data.frame(as.list(se))
   }
-  multigroup <- nrow(se) == 1 | is.list(vc)
+  multigroup <- nrow(se) > 1 | is.list(fsT)
```

Comment:
@winniewytse could you also merge in the changes in the devel branch as now we use `se_fs` for the argument? Thank you.